jest.integration.setup.js

```diff
@@ -19,18 +19,26 @@ beforeAll(() => {
 });
 
 afterAll(() => {
+  console.info("Starting afterAll...");
   const versions = instrumentationsVersionManager.getInstrumantaionsVersions();
-  Object.keys(versions).forEach((lib) => {
-    // updated supported versions file
-    const TESTED_VERSIONS_PATH = `./src/instrumentations/${lib}/tested_versions`;
-    if (!fs.existsSync(TESTED_VERSIONS_PATH)) {
-      fs.mkdirSync(TESTED_VERSIONS_PATH);
-    }
-    const versionStrings = versions[lib].unsupported
-        .map((v) => `!${v}`)
-        .concat(versions[lib].supported)
-        .sort((v1, v2) => semver.compare(v1.replace('!', ''), v2.replace('!', '')))
-        .join('\n');
-    fs.writeFileSync(`${TESTED_VERSIONS_PATH}/${lib}`, versionStrings);
-  });
+  const versions_keys = Object.keys(versions);
+  if (versions_keys.length){
```

Comment:
is this the only change here?

Reply:
yes, especially adding the log for no version to update